Certification standard/main.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <title>Cardano Certification Level Standards</title>
```

Comment:
Suggest changing the title to _Cardano Smart Contract Certification Standard_

Rationale: 

1. more specific about what is being certified
2. title need not talk about levels: that's a second-level detail

Reply:
Changed to suggestion.